src/pages/backOffice/Users/Users.tsx

```diff
@@ -1,8 +1,14 @@
 import * as React from 'react'
 import { UsersProps } from './types'
+import UsersTable from '../../../components/Layout/UsersTable'
 
 const Users: React.SFC<UsersProps> = () => {
-  return <h1>This is my Users</h1>
+  return (
+    <React.Fragment>
+      <h1>This is my Users</h1>
```

Comment:
Replace this with a nicer title and use the `Typography` tag from material ui instead